irc-client-tls.opam

```diff
@@ -17,7 +17,7 @@ depends: [
   "dune" {build}
   "irc-client"
   "lwt"
-  "tls"
+  "lwt_ssl"
```

Comment:
this change looks wrong to me

Reply:
good catch, thanks @hannesm. I first thought of replacing `irc-client-tls`, but in the end, why not just have both? :woman_shrugging: